app/controllers/bonus_credits_controller.rb

```diff
@@ -3,17 +3,20 @@ class BonusCreditsController < ApplicationController
   before_action :authenticate_control!
 
   def index
-    @bonus_credits = BonusCredit.where(game: current_game).order(:round, :team_name, :amount)
+    @bonus_credits = BonusCredit.where(game: current_game).order(:round, :team_id, :amount)
   end
 
   def new
     @round = current_game.round
-    @countries = Game::COUNTRIES
     @bonus_credits = BonusCredit.new
+    @teams = Team.all
```

Comment:
FWIW (partially note to self): The idea of replacing teams with other teams won't work right out of the gate because we also would need to add flags for those teams.